rootfs/opt/fluentd/sbin/boot

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
-FLUENTD_CONF="/opt/fluentd/conf/fluentd.conf"
+FLUENTD_CONF=${FLUENTD_CONF:-"/opt/fluentd/conf/fluentd.conf"}
+INSTALL_BUILD_TOOLS=${INSTALL_BUILD_TOOLS:-"false"}
+BUILD_TOOLS='g++ gcc make ruby-dev'
```

Comment:
You could check how big it is - if it is in the 100-200+ range then I'd be hestitent